### DISCUSSION TITLE
about disabling math operators

### DISCUSSION THREAD

**Discussion by Masber** [mitsuhiko/minijinja#507] [Q&A] [answered]
Dear minijinja community,

I would like to use dashes inside my place holders like this `{{my-var.my-value}}` the problem is that this will be processed as a math operator and panic https://jinja.palletsprojects.com/en/3.0.x/templates/#math.

Since the official documentation says:

> Jinja allows you to calculate with values. This is rarely useful in templates but exists for completeness’ sake

I am wondering if it would it be possible to disable the operators when rendering a jinja file?

thank you very much

**Comment by mitsuhiko** [comment, marked as answer]
Even if you were to be able to disable math operators, they would still not be valid variables.  The syntax cannot be extended for this easily and I also don't think it should be.
